examples/experimental/dagster-airlift/examples/many-dags/many_dags/airflow_dags/migration_state/dag_0.yaml

```diff
@@ -0,0 +1,21 @@
+tasks:
```

Comment:
Hmmm checking all these in seems overkill. Can we make them in-memory and programmatically generated? Or are you using this for manual testing?